packages/nodes-base/package.json

```diff
@@ -16,13 +16,17 @@
   "types": "dist/src/index.d.ts",
   "scripts": {
     "dev": "npm run watch",
-    "build": "tsc && gulp build:icons && gulp build:translations",
+    "build": "npm-run-all build:esbuild build:assets build:dts",
+    "build:esbuild": "node scripts/build.mjs",
+    "build:assets": "gulp build:assets build:translations",
+    "build:dts": "tsc --project tsconfig.build.json",
```

Comment:
The type `NodeVersionedType` is consumed only once and only in `cli/CredentialsHelper.ts`. Maybe we should relocate this class to the package where it is used and thereby also skip type declaration emitting? (Out of scope, tech debt.)

Reply:
Agreed, would be for the best. :)